{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceException(Exception Ex, Boolean FirstTime) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 67\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"4H249-XJ8XZ","FechaInicio":"2024-03-08T13:11:02.6215303-03:00","FechaFin":"2024-03-08T13:11:02.6215303-03:00","Tiempo":0.0},"Information":null},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceInformation(String Origen, String Information) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 82\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"4H249-XJ8XZ","FechaInicio":"2024-03-08T13:11:02.8567169-03:00","FechaFin":"2024-03-08T13:11:02.8567169-03:00","Tiempo":0.0},"Information":null},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceInformation(String Origen, String Information) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 82\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"4H249-XJ8XZ","FechaInicio":"2024-03-08T13:11:02.8567169-03:00","FechaFin":"2024-03-08T13:11:02.8567169-03:00","Tiempo":0.0},"Information":{"Mensaje":"null","Origen":"Autorizacion denegada","Id":0,"IDTransaction":"4H249-XJ8XZ","FechaInicio":"2024-03-08T13:11:02.8572241-03:00","FechaFin":"2024-03-08T13:11:02.8572241-03:00","Tiempo":0.0}},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceException(Exception Ex, Boolean FirstTime) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 67\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"Y2XY7-W2659","FechaInicio":"2024-03-08T13:11:59.4985116-03:00","FechaFin":"2024-03-08T13:11:59.4985116-03:00","Tiempo":0.0},"Information":null},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceInformation(String Origen, String Information) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 82\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"Y2XY7-W2659","FechaInicio":"2024-03-08T13:11:59.886275-03:00","FechaFin":"2024-03-08T13:11:59.886275-03:00","Tiempo":0.0},"Information":null},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceInformation(String Origen, String Information) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 82\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"Y2XY7-W2659","FechaInicio":"2024-03-08T13:11:59.886275-03:00","FechaFin":"2024-03-08T13:11:59.886275-03:00","Tiempo":0.0},"Information":{"Mensaje":"null","Origen":"Autorizacion denegada","Id":0,"IDTransaction":"Y2XY7-W2659","FechaInicio":"2024-03-08T13:12:07.1444821-03:00","FechaFin":"2024-03-08T13:12:07.1444821-03:00","Tiempo":0.0}},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceException(Exception Ex, Boolean FirstTime) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 67\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"48JJ4-YX958","FechaInicio":"2024-03-08T13:12:17.0699895-03:00","FechaFin":"2024-03-08T13:12:17.0699895-03:00","Tiempo":0.0},"Information":null},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceInformation(String Origen, String Information) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 82\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"48JJ4-YX958","FechaInicio":"2024-03-08T13:12:17.0710021-03:00","FechaFin":"2024-03-08T13:12:17.0710021-03:00","Tiempo":0.0},"Information":null},
{"Transaction":null,"Excepction":{"Mensaje":"The underlying provider failed on Open.","Exception":"{\"ClassName\":\"System.Data.Entity.Core.EntityException\",\"Message\":\"The underlying provider failed on Open.\",\"Data\":null,\"InnerException\":{\"Errors\":[{\"Source\":\".Net SqlClient Data Provider\",\"Number\":53,\"State\":0,\"Class\":20,\"Server\":\"\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Procedure\":null,\"LineNumber\":0}],\"ClientConnectionId\":\"00000000-0000-0000-0000-000000000000\",\"ClassName\":\"System.Data.SqlClient.SqlException\",\"Message\":\"Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server)\",\"Data\":{\"HelpLink.ProdName\":\"Microsoft SQL Server\",\"HelpLink.EvtSrc\":\"MSSQLServer\",\"HelpLink.EvtID\":\"53\",\"HelpLink.BaseHelpUrl\":\"http://go.microsoft.com/fwlink\",\"HelpLink.LinkId\":\"20476\"},\"InnerException\":{\"NativeErrorCode\":53,\"ClassName\":\"System.ComponentModel.Win32Exception\",\"Message\":\"No se ha encontrado la ruta de acceso de la red\",\"Data\":null,\"InnerException\":null,\"HelpURL\":null,\"StackTraceString\":null,\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":null,\"HResult\":-2147467259,\"Source\":null,\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\\r\\n   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)\\r\\n   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\\r\\n   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\\r\\n   en System.Data.SqlClient.SqlConnection.Open()\\r\\n   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)\\r\\n   en System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass2_0.<Execute>b__0()\\r\\n   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)\\r\\n   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"1\\n.ctor\\nSystem.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.SqlClient.SqlInternalConnectionTds\\nVoid .ctor(System.Data.ProviderBase.DbConnectionPoolIdentity, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SqlCredential, System.Object, System.String, System.Security.SecureString, Boolean, System.Data.SqlClient.SqlConnectionString, System.Data.SqlClient.SessionData, System.Data.ProviderBase.DbConnectionPool, System.String, Boolean, System.Data.SqlClient.SqlAuthenticationProviderManager)\",\"HResult\":-2146232060,\"Source\":\".Net SqlClient Data Provider\",\"WatsonBuckets\":null},\"HelpURL\":null,\"StackTraceString\":\"   en System.Data.Entity.Core.EntityClient.EntityConnection.Open()\\r\\n   en System.Data.Entity.DbContextTransaction.EnsureOpenConnection()\\r\\n   en System.Data.Entity.DbContextTransaction..ctor(EntityConnection connection)\\r\\n   en System.Data.Entity.Database.BeginTransaction()\\r\\n   en FrameWork.Gestores.Ges_Trace.InitTransaction() en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 40\\r\\n   en FrameWork.Gestores.Ges_Trace.TraceInformation(String Origen, String Information) en P:\\\\Web\\\\Proyectos\\\\Proyectos(Negocios)\\\\Talentosos\\\\Backend\\\\FrameWork2\\\\1.Gestores\\\\Ges_Trace.vb:línea 82\",\"RemoteStackTraceString\":null,\"RemoteStackIndex\":0,\"ExceptionMethod\":\"8\\nOpen\\nEntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089\\nSystem.Data.Entity.Core.EntityClient.EntityConnection\\nVoid Open()\",\"HResult\":-2146233087,\"Source\":\"EntityFramework\",\"WatsonBuckets\":null}","Id":0,"IDTransaction":"48JJ4-YX958","FechaInicio":"2024-03-08T13:12:17.0710021-03:00","FechaFin":"2024-03-08T13:12:17.0710021-03:00","Tiempo":0.0},"Information":{"Mensaje":"null","Origen":"Autorizacion denegada","Id":0,"IDTransaction":"48JJ4-YX958","FechaInicio":"2024-03-08T13:12:17.0715072-03:00","FechaFin":"2024-03-08T13:12:17.0715072-03:00","Tiempo":0.0}},
